### ISSUE TITLE
Sort plugins alphabetically

### ISSUE THREAD

**Issue by Botan626** [JustArchiNET/ASF-ui#1706]
### Checklist

- [X] I read and understood ASF-ui's **[Contributing Guidelines](https://github.com/JustArchiNET/ASF-ui/blob/main/.github/CONTRIBUTING.md)**
- [X] I also read **[FAQ](https://github.com/JustArchiNET/ASF-ui/wiki/FAQ)**
- [X] My idea doesn't duplicate existing ASF-ui functionality described on the **[wiki](https://github.com/JustArchiNET/ASF-ui/wiki)**
- [X] I believe that my idea falls into ASF-ui's scope and should be offered as part of ASF-ui built-in functionality
- [X] This is not **[ASF suggestion](https://github.com/JustArchiNET/ArchiSteamFarm/issues/new/choose)**

This will make reading plugins list easier. Now it's not.

![image](https://github.com/user-attachments/assets/af080710-ebaf-4b61-b5fe-7b966dfa0dc6)

**Comment by Botan626** [comment]
Additionally grouping for built-in and not such plugins will be nice too.

**Comment by Abrynos** [comment]
> built-in

Are you talking about all official plugins or just the ones delivered by default?

**Comment by Botan626** [comment]
> official plugins

> the ones delivered by default

It's not the same thing?

**Comment by Abrynos** [comment]
> It's not the same thing?

no. I recommend you look at ASF main reporitory and checking how many folder names start with `ArchiSteamFarm.OfficialPlugins`.

**Comment by Botan626** [comment]
> no. I recommend you look at ASF main reporitory and checking how many folder names start with `ArchiSteamFarm.OfficialPlugins`.

Without looking I know there are 3 plugins in the `plugins` directory. I called them built-it.
Have a look too just in case:
![image](https://github.com/user-attachments/assets/5bda1557-dc60-4883-a7b2-b9538094eddf)
And they are the ones delivered by default.
So what are you talking about here?

**Comment by Abrynos** [comment]
> `plugins` directory

I said repository.

**Comment by Botan626** [comment]
> I said repository.

And I said built-in plugins. Which is the same as official plugins and the same as plugins delivered by default.

All built-in plugins are official plugins, no matter how many of them come with builds. Take a look at screenshot again.

So again, what are you talking about here?

**Comment by MrBurrBurr** [comment]
@JustArchi Any way for you to add `IsOfficialPlugin` flag to plugins endpoint?

**Comment by Abrynos** [comment]
> [...] same as official plugins and the same as plugins delivered by default

Once again, I'd suggest to take a look at the source repository. In case you're still too lazy, here have a screenshot:
![image](https://github.com/user-attachments/assets/8bab79c2-1cfa-43c0-b42b-77b5b5e57e8d)

> Any way for you to add `IsOfficialPlugin` flag to plugins endpoint?

Anyone could add a boolean with that name to their plugin. Better way of determining it would probably be something like an optional query parameter of `PluginType` (flags) enumeration with values `Official`, `Custom` (and `All` being a combination of the two) 🤔

**Comment by Botan626** [comment]
> Once again, I'd suggest to take a look at the source repository. In case you're still too lazy, here have a screenshot:

Why looking? What for? What you want to say here?

I suggest you to keep looking at the screenshot below and repeat "built-in".
![image](https://github.com/user-attachments/assets/90b26310-0977-41b0-b101-63d1462f0798)

**Comment by Botan626** [comment]
![image](https://github.com/user-attachments/assets/783de82d-0c11-4efb-91e7-33cba2497204)

This is how you gave up on your discussion YOU started?

You came here and started to differentiate between official plugins and the ones delivered by default.
I said they are the same. And they are the same as "biult-in"plugins, regardless of their quantity here and there.

You keep pointing out that official repo has 4 official plugins. AND? What's next?
After 4 replies in my issue you couldn't formulate your thought.

**Comment by Botan626** [comment]
![image](https://github.com/user-attachments/assets/66372a1b-3555-47d1-8f38-4e5a642a6d62)

MonitoringPlugin requires generic ASF, and yet generic ASF build doesn't come with it, so it's not built-in, **_obviously_**.

**Comment by Botan626** [comment]
![image](https://github.com/user-attachments/assets/be5941a4-a2e5-4cd1-adea-bc7848f6fb80)

And I'm telling you again that all built-in plugins, which are delivered by default, are official.

> Are you talking about all official plugins or just the ones delivered by default?

So that Q is just pointless.

**Comment by Botan626** [comment]
![image](https://github.com/user-attachments/assets/ed0a7c96-47b8-4f4b-987f-f54f97446aec)

Initial question should've been: "Do you mean plugins which come in ASF builds when you say `built-in`?" Which are all official.
But then that is a Q with an obvious A. Looks like his professors didn't teach him how to properly formulate his own thoughts.
-
I said 'built-in', but it doesn't mean they should be called such when grouping. Ofc 'official' is the proper name for group.
